src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -498,7 +498,10 @@ private List<string> GetOptionIdsFromFormLayout(string formLayout)
                     Dictionary<string, string> options = new Dictionary<string, string>();
                     foreach (AppOption item in appOptions.Options)
                     {
-                        options.Add(item.Label, item.Value);
+                        if (!options.ContainsKey(item.Label))
```

Comment:
What if the secondary option has data and the first option does not?
Or is there an implied guarantee that that never happens?

Reply:
We do assume that both key and value are identical. I wrote a little bit about this in the bug.

I would consider the rare cases where they are not to be invalid configuration of options.

Reply:
Still not sure we really should "incourage" the hack that is allowing duplicate options... :)

Reply:
This fix is preventing a hard crash right before PDF rendering.